http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -175,15 +175,47 @@ A range of failure status codes is defined by two ASCII `..` full stop character
 
 By default, the body of a request is taken from the `#[payload]` of the incoming Mule message and the response is sent onwards as the `#[payload]` of the output Mule message. You can change this default behavior through the *General > Request > Body* and *General > Output > Target Variable* attributes. Use this attribute to specify a location other than payload for the output data, such as a variable.
 
-== Configure Request Streaming
+== Configure Request Streaming Mode
 
-By default, if the type of the payload is a stream, streaming is used to send the request. You can change this default behavior. Select *General > Request > Request Streaming* with one of the following values:
+When HTTP Connector manages request bodies, the connector considers the type of data to send. You can configure the HTTP *Request* operation to send HTTP requests in chunks. By default, if the payload type is stream, the operation uses the streaming mechanism to send HTTP requests. +
+When streaming, the HTTP request does not contain the `Content-Length` header. The request contains the `Transfer-Encoding` header and sends the body in chunks until the stream is fully consumed.
 
-* *AUTO* (default): The behavior depends on the payload type. If the payload is an InputStream, then streaming is enabled, otherwise it is disabled.
-* *ALWAYS*: Always enable streaming regardless of the payload type.
-* *NEVER*: Never stream, even if the payload is a stream.
+To change the default behavior, configure *Request streaming mode* to any of these options:
 
-When streaming, the request does not contain the `Content-Length` header. It contains the `Transfer-Encoding` header and sends the body in chunks until the stream is fully consumed.
+* *ALWAYS* +
+Always enable streaming regardless of the payload type.
+* *AUTO* (default) +
+The behavior depends on the payload type. If the payload is stream, streaming is enabled, otherwise it is disabled.
+* *NEVER* +
+Never stream, even if the payload is a stream.
+
+The following example shows how to configure  *Request streaming mode* in Studio:
+
+. In the *Mule Palette* view, select *HTTP > Request*.
+. Drag *Request* to the Studio canvas.
+. Set *Path* to `/path`.
+. In the HTTP *Request* operation configuration screen, click the *Request* tab.
+. Set *Request streaming mode* to `ALWAYS`.
+
+image::http-request-6.png[HTTP Request configuration window with Response streaming mode field set to ALWAYS]
+
+In the Configuration XML editor, the `requestStreamingMode` configuration looks like this:
```

Comment:
Is Configuration capped because it is part of the editor's name? If it is just a type, lowercase; if a label, use bold formatting.